client/views/edit/edit.html

```diff
@@ -10,7 +10,7 @@ <h1 class="formtitle">Edit</h1>
 		<div class="inputcontainer" style="text-align: center;">
```

Comment:
@singhsanket143 actually, can you fix the formatting in the whole file?

Reply:
@peter-hank Updated the file formatting

Reply:
@singhsanket143 it's still not too great, can you use something like https://codebeautify.org/htmlviewer/ to format it? Thanks.

Reply:
@peter-hank  I used the following tool and have refactored the code. Please review it. Also, shall I use the same tool for refactoring in other branches also?